service/storage_service.cc

```diff
@@ -1357,6 +1358,11 @@ future<> storage_service::raft_initialize_discovery_leader(const join_node_reque
 
     insert_join_request_mutations.emplace_back(co_await _sys_ks.local().make_auth_version_mutation(write_timestamp, db::system_keyspace::auth_version_t::v2));
 
+    auto sl_driver_mutations = co_await _sl_controller.local().get_create_driver_service_level_mutations(_sys_ks.local(), write_timestamp);
```

Comment:
perhaps get_create_driver_service_level_mutations should be static too, if set_service_level_mutations is static now